tic-tac-toe-app/game.js

```diff
@@ -0,0 +1,83 @@
+class Game{
+    constructor(){
+        this.playerOne = new Player()
+        this.playerTwo = new Player()
+        this.playerOne.move = 'X'
+        this.playerTwo.move = 'O'
+        this.board = ['', '', '', '', '', '', '', '', '']
+        this.posTracker = {}
+        this.turn = 1
+        this.winner = null
+    }
+    
+    checkBoard(candidate){
+
+        for(let position in this.board){
+            this.posTracker[position] = this.board[position] 
+        }
+        //Checks each row to see if a player won
+        if(this.posTracker['0'] === this.posTracker['1'] && this.posTracker['0'] === this.posTracker['2'] && this.posTracker['0'] !== ''){
+            candidate % 2 === 0 ? game.playerOne.score += 1 : game.playerTwo.score += 1
+            this.gameOver() 
+        }
```

Comment:
Each of the if statements are essentially the same except for the  `posTracker` position. Consider refactoring in order to reduce the amount of repetitive code here.

Reply:
I'll work on this!